std/net/curl.d

```diff
@@ -3480,6 +3534,66 @@ struct FTP
     {
         p.curl.set(CurlOption.infilesize_large, to!curl_off_t(len));
     }
+
+    /**
+     * Get various timings defined in $(LREF etc.c.curl.CurlInfo).
+     * The value is usable only if the return value is equal to $(LREF CurlError.ok).
```

Comment:
Broken link, as `CurlError` is not defined in the same file. Should be `$(REF ok, etc, c, curl, CurlError)`